central/globaldb/postgres.go

```diff
@@ -57,6 +57,8 @@ SELECT TABLE_NAME
 ) a;`
 
 	versionQuery = `SHOW server_version;`
+
+	connectionQuery = `SELECT datname, COUNT(datid) FROM pg_stat_activity WHERE state <> 'idle' AND datname IS NOT NULL GROUP BY datname;`
```

Comment:
Probably makes sense to exclude current connection where this query is executed?

```
where pid != pg_backend_pid()
```

Also would be quite useful to know max_connections together with this metric, is it exposed somewhere? Otherwise in the same query `current_setting('max_connections')::int`.

Reply:
I left the active one as is.  Figure if we excluded the one making the request we could easily forget that it counts as we approach the limit.  I did add a similar one for total that would include active and idle.  And then as suggested an overarching one for the max connections allowed.

Reply:
> Figure if we excluded the one making the request we could easily forget that it counts as we approach the limit. 

Fair enough 👍